<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editPage</class>
 <widget class="QWizardPage" name="editPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="idlineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="namelineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="scorelineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="ranklineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="calculatepushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>换算</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>38</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>学号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>成绩：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>名次：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="umbrellaLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icon.qrc">:/images/umbrella.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>idlineEdit</tabstop>
  <tabstop>namelineEdit</tabstop>
  <tabstop>scorelineEdit</tabstop>
  <tabstop>calculatepushButton</tabstop>
  <tabstop>ranklineEdit</tabstop>
  <tabstop>deleteButton</tabstop>
 </tabstops>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>editPage</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
